policies/pol/sample_paloalto.pol

```diff
@@ -0,0 +1,48 @@
+################
```

Comment:
Added, as well as the expected output in the test folder.
Palo alto comes with a wide array of existing "Applications", that's why I made it a simple variable that the user can define.